src/main/java/com/shaft/tools/io/ReportManager.java

```diff
@@ -406,7 +406,7 @@ public static String getTestMethodName() {
 
     public static void setTestCaseName(String scenarioName) {
         Allure.getLifecycle().updateTestCase(testResult -> testResult.setName(scenarioName));
-        if (!"".equals(featureName)) {
+        if (!featureName.equals("")) {
```

Comment:
Codacy found an issue: [Position literals first in String comparisons](https://app.codacy.com/manual/mohab.mohieeldeen/SHAFT_ENGINE/pullRequest?prid=5739304)

Reply:
fixed